lancero_source.go

```diff
@@ -167,7 +181,7 @@ func (ls *LanceroSource) ConfigureMixFraction(processorIndex int, mixFraction fl
 	go func() {
 		ls.runMutex.Lock()
 		defer ls.runMutex.Unlock()
-		ls.Mix[processorIndex].mixFraction = mixFraction
+		ls.Mix[processorIndex].mixFraction = mixFraction / float64(ls.nsamp)
```

Comment:
What about renaming `Mix.mixFraction` to `Mix.mixFractionOverNsamp` so we don't have the case that numbers with different value have the same name. Meaning, the user sends `MixFraction` over the RCP server, so anything called `MixFraction` should have the value they sent.

Reply:
Hmm. I see your point. Maybe I can think of something more terse. Perhaps `scaledMixFraction` or `mixRatio` or `mixMultiplier` or better, leave "mix" out of it: `errorMultiplier`.

Okay, I'll make the proposed revisions and pull.

Reply:
I prefer the explicit name, it's a form of self documentation. The value is only used in a few spots, terseness won't buy you much.